File: WanaD.WebClient/app/src/Modules/Core/Scripts/AppServiceContract/FileRelationData.ts

 

 

 

    import FileInfoData from 'Modules/Core/Scripts/AppServiceContract/FileInfoData';

 
class FileRelationData 
{
    
        Id: string | null;
    
        ReferenceId: string | null;
    
        FileInfoId: string;
    
        RelationType: number;
    
        CreatorUserProfileId: string | null;
    
        FileInfoData: FileInfoData;
    
        IsUploadByBU: boolean;
    
        IsUploadByReviewer: boolean;
    
        Ordinal: number | null;
    
}
 
export default FileRelationData;
 

File: Fs.Erp.Core.AppServiceContract/FileRelationData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Fs.Erp.Core.AppServiceContract
{
    public class FileRelationData
    {
        public FileRelationData()
        {
            FileInfoData = new FileInfoData();
        }
        public Guid? Id { get; set; }
        public Guid? ReferenceId { get; set; }
        public Guid FileInfoId { get; set; }
        public int RelationType { get; set; }
        public Guid? CreatorUserProfileId { get; set; }
        public FileInfoData FileInfoData { get; set; }
        public bool IsUploadByBU { get; set; }
        public bool IsUploadByReviewer { get; set; }
        public int? Ordinal { get; set; }
    }
}
